RNWCPP/ReactUWP/Views/ScrollViewManager.cpp

```diff
@@ -77,6 +77,8 @@ folly::dynamic ScrollViewManager::GetExportedCustomDirectEventTypeConstants() co
 XamlView ScrollViewManager::CreateViewCore(int64_t tag)
 {
   winrt::ScrollViewer scrollViewer;
+  scrollViewer.HorizontalScrollBarVisibility(winrt::ScrollBarVisibility::Visible);
+  scrollViewer.VerticalScrollBarVisibility(winrt::ScrollBarVisibility::Visible);
```

Comment:
This will cause both scrollbars to always be visible.  Should we use Auto instead?

Reply:
The documentation says the Default for both is Visible.
It is possible that their Visible behaves differently then our Visible though... I should check.